ANMN/temperature_grid_product/anmn_temp_gridded_product.py

```diff
@@ -397,45 +312,45 @@ def generate_fv02_netcdf(df, nc_file_list):
             setattr(output_netcdf_obj[var], var_att, getattr(input_netcdf_obj[var], var_att))
 
-    add_var_att_from_input_nc_to_output_nc('DEPTH')
-    add_var_att_from_input_nc_to_output_nc('LONGITUDE')
+    add_var_att_from_input_nc_to_output_nc('TIME')
     add_var_att_from_input_nc_to_output_nc('LATITUDE')
+    add_var_att_from_input_nc_to_output_nc('LONGITUDE')
+    add_var_att_from_input_nc_to_output_nc('DEPTH')
+    add_var_att_from_input_nc_to_output_nc('TEMP')
 
-    # convert datetime64 to datetime
-    time_values      = transform_datetime64_datetime(df.columns.values)
-    time_val_dateobj = date2num(time_values, output_netcdf_obj['TIME'].units, output_netcdf_obj['TIME'].calendar)
+    time_val_dateobj = date2num(time_1d_interp, var_time.units, var_time.calendar)
     var_time[:]      = time_val_dateobj
 
-    output_netcdf_obj.time_coverage_start = min(time_values).strftime('%Y-%m-%dT%H:%M:%SZ')
-    output_netcdf_obj.time_coverage_end   = max(time_values).strftime('%Y-%m-%dT%H:%M:%SZ')
+    output_netcdf_obj.time_coverage_start = min(time_1d_interp).strftime('%Y-%m-%dT%H:%M:%SZ')
+    output_netcdf_obj.time_coverage_end   = max(time_1d_interp).strftime('%Y-%m-%dT%H:%M:%SZ')
 
-    output_netcdf_obj.geospatial_vertical_min = float(df.index.values.min())
-    output_netcdf_obj.geospatial_vertical_max = float(df.index.values.max())
+    output_netcdf_obj.geospatial_vertical_min = float(np.min(depth_1d_interp))
+    output_netcdf_obj.geospatial_vertical_max = float(np.max(depth_1d_interp))
 
     abstract = ("This product aggregates Temperature logger data collected at "
                 "these nominal depths (%s) on the mooring line during the %s "
                 "deployment by averaging them temporally and interpolating them "
-                "vertically on a common grid. The grid covers from %s to %s "
+                "vertically at consistent depths. The grid covers from %s to %s "
                 "temporally and from %s to %s metres vertically. A cell is %s "
-                "minutes wide and %s metre high") % (instrument_nominal_depth,
+                "minutes wide and %s metre high.") % (", ".join(map(str, instrument_nominal_depth)),
                                                 output_netcdf_obj.deployment_code,
                                                 output_netcdf_obj.time_coverage_start,
                                                 output_netcdf_obj.time_coverage_end,
-                                                df.index.values.min(),
-                                                df.index.values.max(),
+                                                output_netcdf_obj.geospatial_vertical_min,
+                                                output_netcdf_obj.geospatial_vertical_max,
                                                 output_netcdf_obj.temporal_resolution,
                                                 output_netcdf_obj.vertical_resolution)
```

Comment:
I don't see the need to duplicate all this metadata in the abstract, especially temprao/spatial ranges.

Reply:
Fair enough. I suggest we retain the abstract from original FV01 file and add general information about this FV02 product, yet still including temporal and vertical resolution info.